docker/Dockerfile

```diff
@@ -2,7 +2,8 @@ FROM adoptopenjdk/openjdk11:alpine-slim as builder
 LABEL maintainer="wout.slakhorst@nuts.nl"
 RUN mkdir /opt/nuts
 COPY / /opt/nuts/
-RUN cd /opt/nuts && ./gradlew bootJar
+WORKDIR /opt/nuts
+RUN ./gradlew bootJar
```

Comment:
If you want your application config to be mountable, I think you have to pass the config location as param